custom_model_runner/datarobot_drum/drum/language_predictors/r_predictor/score.R

```diff
@@ -55,14 +56,18 @@ has_read_input_data_hook <- function() {
 #' @export
 #'
 #' @examples
-load_serialized_model <- function(model_dir) {
+load_serialized_model <- function(model_dir, target_type) {
     model <- NULL
     if (!isFALSE(load_model_hook)) {
         model <- load_model_hook(model_dir)
     }
     if (is.null(model)) {
         file_names <- dir(model_dir, pattern = CUSTOM_MODEL_FILE_EXTENSION)
         if (length(file_names) == 0) {
+            # Allow no serialized models if it is a transform
```

Comment:
is this to support hardcoded transforms?

Reply:
Yes - transforms with no model artifact